resources/aws/internet_gateway.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/giantswarm/micrologger"
 )
 
-type Gateway struct {
+type InternetGateway struct {
```

Comment:
Gateway needs to be renamed to InternetGateway so we can also have a NatGateway resource. This is all just renaming.

Reply:
Do you talk about resources as in operatorkit framework resources? Will we need to do this gateway magic with Cloudformation as well?

Reply:
This is still the old style resources using the Go SDK. For the operatorkit migration I think we need to create a legacy resource. We can then put the current resources/aws package inside it.

We can then have an operatorkit resource for the Cloud Formation stacks and start migrating to CF.